build/src/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const {spawn} = require('child_process');
+const app = express();
+const port = 8000;
+app.use(bodyParser.json());
+app.use(bodyParser.text());
+
+app.get('/address/:address', (req, res) => {
+    
+    if (req.params.address.length != 42)
```

Comment:
I might be forgiving here and accept both 40 character and 42 character strings. I would then remove leading '0x' if present. That's just my preference. I'm sure that's bad form.